File: Learning/OtherLanguages/C/C练习题/Use C/18.c
#include <stdio.h>

void main()
{
    int i, a, n, sum, temp;

    printf("Input a and n: ");
    scanf("%d%d", &a, &n);

    temp = a;
    sum = a;
    printf("sum=%d", a);
    for(i=1; i<n; i++)
    {
        a = a*10 + temp;
        sum += a;
        printf("+%d", a);
    }

    printf("=%d\n", sum);
}

File: Learning/OtherLanguages/C/C练习题/Use Python/18.py

a = input('请输入一个数字(1-9)：')
n = int(input('请输入相加的个数：'))

an = int(a * n)

s = (int(a) + an) * n // 2

print('s = ', end='')

for i in range(1, n):
    print(a * i, '+ ', end='')
    
print(a * n, '=', s)

